imports/plugins/included/product-variant/client/templates/products/productGrid/content.js

```diff
@@ -7,5 +7,12 @@ Template.gridContent.helpers({
     if (this.price && this.price.range) {
       return this.price.range;
     }
+  },
+  ratings: function (rating) {
+    const countArr = [];
```

Comment:
you can use Array functions here instead of a for loop
e.g. `Array(5).fill({})` returns new Array with 5 {} elements